lib/ansible/module_utils/basic.py

```diff
@@ -466,6 +468,72 @@ def heuristic_log_sanitize(data, no_log_values=None):
         output = remove_values(output, no_log_values)
     return output
 
+def bytes_to_human(size, isbits=False, unit=None):
+
+    base = 'Bytes'
+    if isbits:
+        base = 'bits'
+    suffix = ''
+
+    for suffix, limit in sorted(SIZE_RANGES.iteritems(), key=lambda item: -item[1]):
```

Comment:
need to change this for python3 compatibility.  We've imported the iteritems function from six earlier so we can just do this::
``` python
    for suffix, limit in sorted(iteritems(SIZE_RANGES), key=lambda item: -item[1]):